src/components/pages/ImageClassification/sampleData/image_endpoints.md

```
@@ -0,0 +1,1455 @@
+      "status": 3,
+      "message": null,
+      "data": {}
+    }
+  ]
+}
+```
+
+### get all images in project
+- use `exclude` query param to greatly reduce response size.
+- example:
+  - GET https://dev-api.datamermaid.org/v1/projects/df2bb123-5bfc-4f3f-aa88-bb102d3b2b35/classification/images/?exclude=data,points,created_by,updated_by,created_on,updated_on,original_image_width,original_image_height,location,comments,image,photo_timestamp
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)